src/pages/Room/components/BannerSlider.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Navigation } from 'swiper';
+import 'swiper/css';
+import 'swiper/css/navigation';
+
+const BannerSlider = ({
+  name,
+  preson,
+  max_preson,
+  area,
+  room_images,
+  description,
+  bed,
+  price,
+  checkInData,
+  checkOutData,
+}) => {
+  const BookingDays = checkInData?.daysLeft - checkOutData?.daysLeft;
+  return (
+    <BannerSliderLayout>
+      <InfoWrapper>
+        <RoomInfoTitle>ROOM INFORMATION</RoomInfoTitle>
+        <RoomName>{name}</RoomName>
+        <RoomPrice>
+          {checkOutData ? `₩ ${BookingDays * price} / ${BookingDays}박` : '-'}
+        </RoomPrice>
+        <RoomDesc>{description}</RoomDesc>
+        <RoomInfoEtc>
+          <RoomOptions>체크인 15:00 / 체크아웃 11:00</RoomOptions>
+          <RoomOptions>
+            기준 인원 {preson} (최대 인원 {max_preson})
+          </RoomOptions>
+          <RoomOptions>객실면적 {area}㎡</RoomOptions>
+          <RoomOptions>침대 {bed}</RoomOptions>
+        </RoomInfoEtc>
+      </InfoWrapper>
+      <BannerWrapper>
+        <StyledSwiper
+          className="banner"
+          spaceBetween={0}
+          slidesPerView={1}
+          navigation
+          loop={true}
+          modules={[Navigation]}
+        >
+          {room_images?.map((image, idx) => {
+            return (
+              <SwiperSlide key={idx}>
+                <RoomImage src={image} alt="roomImages" />
+              </SwiperSlide>
+            );
+          })}
+        </StyledSwiper>
+      </BannerWrapper>
+    </BannerSliderLayout>
+  );
+};
+
+export default BannerSlider;
+
+const BannerSliderLayout = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 1290px;
+  margin: 50px auto 0;
+`;
+
+const BannerWrapper = styled.div`
+  width: 960px;
+  height: 540px;
+`;
+
+const InfoWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const RoomInfoTitle = styled.p`
+  font-size: 12px;
+  letter-spacing: 3.6px;
+`;
+
+const RoomName = styled.div`
+  margin-top: 20px;
+  font-size: 30px;
+  line-height: 1.2;
+`;
+
+const RoomPrice = styled.p`
+  font-size: 18px;
+  margin-top: 25px;
+`;
+
+const RoomDesc = styled.div`
+  font-size: 14px;
+  color: #333;
+  line-height: 1.7;
+  margin-top: 30px;
+  padding-right: 20px;
+  padding-bottom: 30px;
```

Comment:
```suggestion
  margin-top: 30px;
  padding-right: 20px;
  padding-bottom: 30px;
    font-size: 14px;
  color: #333;
  line-height: 1.7;
```

* css 속성 순서대로 수정 해 주세요~
* 다른 속성들도 확인해서 수정 해 주세요!